payments-core/AndroidManifest.xml

```diff
@@ -65,6 +65,10 @@
         <activity
             android:name=".googlepaylauncher.GooglePayPaymentMethodLauncherActivity"
             android:theme="@style/StripeGooglePayDefaultTheme" />
+
+        <activity android:name=".payments.paymentlauncher.PaymentLauncherHostActivity"
+            android:theme="@style/StripeDefaultTheme" />
```

Comment:
This could be named `PaymentConfirmationActivity` or something along those lines. In any case, I don't think we can drop the `Host` part of the name - I don't think it adds any meaning.

Also add `exported=false`.

Reply:
Renamed to `PaymentLauncherComfirmationActivity`